live_score_flutter_app/lib/widgets/ongoing_game_card.dart

```diff
@@ -1,45 +1,48 @@
 import 'package:flutter/material.dart';
 
+import '../models/game.dart';
+
 class OngoingGameCard extends StatelessWidget {
-  String gameType;
-  String nameTeam1;
-  String nameTeam2;
-  String scoreTeam1;
-  String scoreTeam2;
-  OngoingGameCard(
-      {required this.gameType,
-      required this.nameTeam1,
-      required this.nameTeam2,
-      required this.scoreTeam1,
-      required this.scoreTeam2});
+  Game game;
+  VoidCallback? onPressed;
+  OngoingGameCard({required this.game,this.onPressed});
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      padding: const EdgeInsets.all(16),
-      margin: const EdgeInsets.symmetric(vertical: 16),
-      height: 200,
-      width: 0.75*MediaQuery.of(context).size.width,
-      decoration: BoxDecoration(border: Border.all()),
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-        Text(gameType),
-        const SizedBox(
-          height:30,
-        ),
-        Row(
-          mainAxisAlignment: MainAxisAlignment.spaceAround,
-          children: [
-            Column(
-              children: [Text(nameTeam1),const SizedBox(height:20), Text(scoreTeam1)],
-            ),
-            Column(
-              children: [Text(nameTeam2),const SizedBox(height:20), Text(scoreTeam2)],
-            )
-          ],
-        ),
-      ]),
+    return GestureDetector(
+      onTap: onPressed,
+      child: Container(
+        padding: const EdgeInsets.all(16),
+        margin: const EdgeInsets.symmetric(vertical: 16),
+        height: 200,
+        width: 0.75 * MediaQuery.of(context).size.width,
+        decoration: BoxDecoration(border: Border.all()),
+        child: Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
+          Text(game.gameType),
+          const SizedBox(
+            height: 30,
+          ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceAround,
+            children: [
+              Column(
+                children: [
+                  Text(game.team1),
+                  const SizedBox(height: 20),
+                  Text(game.score1.toString())
+                ],
+              ),
+              Column(
+                children: [
+                  Text(game.team2),
+                  const SizedBox(height: 20),
+                  Text(game.score2.toString())
+                ],
+              )
+            ],
+          ),
+        ]),
+      ),
     );
   }
 }
```

Comment:
also please add the google-services.json file in .gitignore and .env file too if you wished to change the firebase_option.dart file too.

Reply:
Ok I will